modules/carto/src/layers/carto-bqtiler-layer.js

```diff
@@ -0,0 +1,25 @@
+import CartoLayer from './carto-layer';
+
+const BQ_TILEJSON_ENDPOINT = 'https://us-central1-cartobq.cloudfunctions.net/tilejson';
+
+const defaultProps = {
+  ...CartoLayer.defaultProps
```

Comment:
This is unnecessary. `defaultProps` is automatically merged with that of the parent class.